app/Api/Controllers/APIController.php

```diff
@@ -10,17 +10,49 @@
 
 class ApiController extends Controller
 {
-    public function __construct() {
-
-    }
-
     /**
     * Get info about the install
     */
-    public function get_info() {
+    public function get_info()
+    {
         $versions['git'] = `git rev-parse --short HEAD`;
         $versions['db_schema'] = DB::select('SELECT `version` FROM `dbSchema` LIMIT 1')[0]->version;
+        $versions['php'] = phpversion();
+        $versions['mysql'] = DB::select('SELECT version() AS version')[0]->version;
+        if (empty($_SERVER['SERVER_SOFTWARE'])) {
+            $_SERVER['SERVER_SOFTWARE'] = "";
+        }
+        $versions['apache'] = str_replace('Apache/', '', $_SERVER['SERVER_SOFTWARE']);
         return $versions;
     }
 
+    /**
+     * Get statistics about the install
+    **/
+    public function get_stats()
+    {
+        $stats['devices']    = DB::select('SELECT COUNT(device_id) AS `total` FROM `devices`')[0]->total;
```

Comment:
This can be changed too Device::all()->count();
And the same for ports and users.